Warp.WebApp/Models/AppFile.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using System.ComponentModel.DataAnnotations;
+
+namespace Warp.WebApp.Models
+{
+    public record struct AppFile
+    {
+        public int Id { get; set; }
+
+        public byte[] Content { get; set; }
+
+        [Display(Name = "File Name")]
```

Comment:
`DisplayAttribute` does not needed for this file, because we use API controller to return results